server_ui/advertise.go

```diff
@@ -100,11 +100,16 @@ func advertiseInternal(ctx context.Context, server server_utils.XRootDServer) er
 	if err != nil {
 		return errors.Wrap(err, fmt.Sprintf("%s failed to get namespaceAds from the director", server.GetServerType()))
 	}
+	serverUrl := param.Origin_Url.GetString()
+	webUrl := param.Server_ExternalWebUrl.GetString()
+	prefix := param.Origin_NamespacePrefix.GetString()
+	if server.GetServerType().IsEnabled(config.CacheType) {
+		serverUrl = param.Origin_Url.GetString()
```

Comment:
Should this be `Origin_Url`? We just set that above, but this looks conditional to caches.